features/promotion/deleting_promotion.feature

```diff
@@ -0,0 +1,15 @@
+@promotion
+Feature: Deleting a promotion
+    In order to delete test or invalid promotions
```

Comment:
We agreed on one version:
`In order to remove test, obsolete or incorrect {resource}`